src/interp_routines.F

```diff
@@ -566,8 +577,11 @@ subroutine vinterp_flx5(stag,ix,jy,kz,c1,c2,rrw,dumz,a)
           dumz(i,j,k) = flx5(a(i,j,k+2),a(i,j,k+1),a(i,j,k  ),a(i,j,k-1),a(i,j,k-2))
         endif
       enddo
+    ENDDO
 
-      k = nk-1
+    k = nk-1
+    !$acc parallel loop gang vector collapse(2) default(present)
```

Comment:
Do we need OpenMP directive here?

Reply:
Yes we do, I have added in OpenMP directives.